src/background/utils.js

```diff
@@ -28,8 +29,8 @@ export const getTippingContractInstance = async (tx) => {
 };
 
 export const contractCallStatic = async ({ tx, callType }) => {
-  const { account } = store.getters;
-  if (typeof callType !== 'undefined' && callType === 'static' && account) {
+  const { isLoggedIn } = useAccounts({ store });
+  if (typeof callType !== 'undefined' && callType === 'static' && isLoggedIn.value) {
```

Comment:
You could store `typeof callType !== 'undefined' && callType === 'static'` in a variable as it's repeated here

Reply:
Makes sense. But I noticed that this is repetition makes no sense in general because the only difference between those two checks is determining if user is logged in. So I changed the structure to make it more readable.